### PR TITLE
Launch point for business modal- dev collab only

### PR DESCRIPTION
**Pull request by denamwangi**
For collaboration only

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationProjectsDashboard/projectNav.jsx [hunk 1]
```diff
@@ -36,8 +37,22 @@ const ProjectNav = createReactClass({
...
         onSelect: () => {
-          openCreateTeamModal({
+          // openCreateTeamModal({
+          //   organization: org,
+          //   onClose: data => {
+          //     if (!data) return;
+
+          //     browserHistory.push(`/settings/${org.slug}/teams/${data.slug}/members/`);
+          //   },
+          // });
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/organizationProjectsDashboard/projectNav.jsx]: Could this be deleted instead of commented out?

Comment (denamwangi) [reply src/sentry/static/sentry/app/views/organizationProjectsDashboard/projectNav.jsx]: Hey Mark- it's a temporary quick & dirty branch for dev collaboration only. It won't ever be merged. Thanks for hopping on though.

Comment (benvinegar) [reply src/sentry/static/sentry/app/views/organizationProjectsDashboard/projectNav.jsx]: > It won't ever be merged.

Did this require a PR? What if we just pushed the branch?

Comment (denamwangi) [reply src/sentry/static/sentry/app/views/organizationProjectsDashboard/projectNav.jsx]: next time def

**Pull request closed without merging**